### PR TITLE
issue #30 - update contributing guidelines and rename file

### PR DESCRIPTION
**Pull request by ruqoyyasadiq**
As per the issue #30 this PR contains work on renaming the contributing guidelines file to `contributing.md`. It also features updates to the file to reflect the easy accessibility to the issues and projects pages.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 
-Want to join the Contributors' channel? 
+Want to join the Contributors' channel?
 
 First Sign up [here](https://devcenter-square-slack.herokuapp.com/) to join the DC Square Slack Channel, then fill this [contributors form](https://docs.google.com/forms/d/1JiRNzYS69ojHPCql254HMMmp72cRb2TMGVC6BiOTw6E/viewform?c=0&w=1) to get added to the the Disease Info Contributors' channel.
 
 **The #1 contribution you could make is to blog, share, post, tweet, and tell people about disease-info.  This will go a long way in building a sustainable community.**
 
-We are happy to accept contributions of any kind, including feedback and ideas, translations for other locales, and functionality. 
+Your next stop would be at the [issues page](https://github.com/devcenter-square/disease-info/issues). Here can find a list of issues to fixed as regards to the ongoing project.
```
Comment (osioke) [inline CONTRIBUTING.md]: Really nice work @andela-rsadiq 👍 

See this typo on line 12 -> ...Here _you_ can find a list of issues to _fix_ as regards _either the_ ongoing project, _or bugs on the app._

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
-Want to join the Contributors' channel? 
+Want to join the Contributors' channel?
 
 First Sign up [here](https://devcenter-square-slack.herokuapp.com/) to join the DC Square Slack Channel, then fill this [contributors form](https://docs.google.com/forms/d/1JiRNzYS69ojHPCql254HMMmp72cRb2TMGVC6BiOTw6E/viewform?c=0&w=1) to get added to the the Disease Info Contributors' channel.
 
 **The #1 contribution you could make is to blog, share, post, tweet, and tell people about disease-info.  This will go a long way in building a sustainable community.**
 
-We are happy to accept contributions of any kind, including feedback and ideas, translations for other locales, and functionality. 
+Your next stop would be at the [issues page](https://github.com/devcenter-square/disease-info/issues). Here you can find a list of issues to fix as regards either the ongoing project, or bugs on the app.
+
```
Comment (ruqoyyasadiq) [inline CONTRIBUTING.md:13]: @osioke corrected it here

File: CONTRIBUTING.md (hunk 1 above)
Comment (andela-rsadiq) [inline CONTRIBUTING.md]: corrected!

**Comment by osioke** [comment]
@SundayAdefila @ammezie @kinsomicrote @andela-ysanni I think the doc is good to go, or what do you think?

**Comment by x6iae** [comment]
@andela-rsadiq 👍 Real good work. One more thing needed to close the issue with this PR, I suppose: - 

> mention the issues page and the _repo's current project page_ 

Is this the front-end UI? ( https://disease-info.herokuapp.com ) I need to confirm this first though.

**Comment by x6iae** [comment]
Oh... yeah, above comment doesn't apply. the projects page is well captured as https://github.com/devcenter-square/disease-info/projects . From my end, the PR is good to go!

File: CONTRIBUTING.md (hunk 2 above)
Comment (kinsomicrote) [reply CONTRIBUTING.md:13]: Is `would` the right word to use? I'm not sure, but I think `should` is preferable. Reason: that line sounds like a suggestion, using `would` betrays that. Correct me if I am wrong.
Beside this, I think this is good to go.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
 
-Want to join the Contributors' channel? 
+Want to join the Contributors' channel?
 
 First Sign up [here](https://devcenter-square-slack.herokuapp.com/) to join the DC Square Slack Channel, then fill this [contributors form](https://docs.google.com/forms/d/1JiRNzYS69ojHPCql254HMMmp72cRb2TMGVC6BiOTw6E/viewform?c=0&w=1) to get added to the the Disease Info Contributors' channel.
 
 **The #1 contribution you could make is to blog, share, post, tweet, and tell people about disease-info.  This will go a long way in building a sustainable community.**
 
-We are happy to accept contributions of any kind, including feedback and ideas, translations for other locales, and functionality. 
+Your next stop should be at the [issues page](https://github.com/devcenter-square/disease-info/issues). Here you can find a list of issues to fix as regards either the ongoing project, or bugs on the app.
```
Comment (andela-rsadiq) [inline CONTRIBUTING.md]: fixed @kinsomicrote

**Comment by x6iae** [comment]
Great! merging now.

**Pull request merged**